pyproject.toml

```diff
@@ -19,7 +19,7 @@ pytest-cov = "*"
 pip-audit = "^2.5.5"
 python-semantic-release = "^7.33.4"
 pre-commit = "^3.3.2"
-beaker-pyteal = "^1.0.0"
+beaker-pyteal = "1.1.0b1"
```

Comment:
Is this right?

Reply:
@robdmoore see `further notes` - will change this as soon as devrels publish the beaker 1.1.0